uncover/session.go

```diff
@@ -37,15 +40,44 @@ func NewSession(keys *Keys, retryMax, timeout int) (*Session, error) {
 	client := retryablehttp.NewWithHTTPClient(httpclient, options)
 
 	session := &Session{
-		Client:   client,
-		Keys:     keys,
-		RetryMax: retryMax,
+		Client:     client,
+		Keys:       keys,
+		RetryMax:   retryMax,
+		RateLimits: &ratelimit.MultiLimiter{},
+	}
+
+	var err error
+	rateLimitOpts := &ratelimit.Options{
+		MaxCount:    1,
+		Duration:    time.Duration(rateLimit),
+		IsUnlimited: rateLimit == 0,
+	}
+
```

Comment:
@xm1k3 , this seems like code of old logic which uses `delay` when ratelimit is given ex: `-rl 4` it is 4 req/sec
hence options should be 
```go
                 MaxCount:    rateLimit,
		Duration:    time.Second,
		IsUnlimited: rateLimit == 0,
```

also we need to add `-rlm` which is req/min . 
Ref: nuclei